search-service/src/main/kotlin/com/egm/stellio/search/util/QueryUtils.kt

```diff
@@ -22,29 +22,21 @@ fun parseAndCheckQueryParams(
     val withAudit = hasValueInOptionsParam(
         Optional.ofNullable(queryParams.getFirst(QUERY_PARAM_OPTIONS)), OptionsParamValue.AUDIT
     )
-    val count = queryParams.getFirst(QUERY_PARAM_COUNT)?.toBoolean() ?: false
-    val ids = parseRequestParameter(queryParams.getFirst(QUERY_PARAM_ID)).map { it.toUri() }.toSet()
-    val types = parseAndExpandRequestParameter(queryParams.getFirst(QUERY_PARAM_TYPE), contextLink)
     val temporalQuery = buildTemporalQuery(queryParams, contextLink)
-    val (offset, limit) = extractAndValidatePaginationParameters(
+    val queryParams = parseAndCheckQueryParams(
```

Comment:
this naming is dangerous as you are shadowing a parameter of the function

Reply:
I rename this to `parseAndCheckParams`